lib/httparty/request.rb

```diff
@@ -9,7 +9,8 @@ class Request #:nodoc:
       Net::HTTP::Head,
       Net::HTTP::Options,
       Net::HTTP::Move,
-      Net::HTTP::Copy
+      Net::HTTP::Copy,
+      Net::HTTP::Mkcol
```

Comment:
Had actually thought about it, but decided to avoid in the first place to keep konsistency with the original.

It has been added now in 5f94172 :smiley: